### ISSUE TITLE
conflict of interest review unclear 

### ISSUE THREAD

**Issue by ultrasaurus** [cncf/sig-security#341]
The intent was to only require chair review if there were hard-conflicts or the security assessment facilitator flagged a concern of too many soft conflicts -- this is not at all clear in how we wrote it down and should be fixed.  

In the meantime, let's not block assessments on chair review if @JustinCappos agrees that there are no blockers and the project team is ready to move forward.

https://github.com/cncf/sig-security/issues/308#issuecomment-568119151

**Comment by lumjjb** [comment]
I think when I wrote the text before, my initial thinking was the same as the comment, so perhaps we got lost in translation somewhere. I do agree that probably chairs should not be required to unblock if there are no complications. 

Perhaps we can specify responsibility of the security lead reviewer to also ensure all conflicts and specified before proceeding to review, and to raise to security assessment facilitator if there are any conflicts. And from there its either check off by @JustinCappos or if it judgement says someone should not be on the project then it gets raised up to chairs?
